strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/yaml/configuration/repository/MavenRepositoryConfigurationDto.java

```diff
@@ -74,4 +79,12 @@ public CustomRepositoryConfiguration getImmutable()
     {
         return new MavenRepositoryConfigurationData(this);
     }
+
+    public Set<String> getDigestAlgorithmSet() {
+        return digestAlgorithmSet;
+    }
+
+    public void setDigestAlgorithmSet(Set<String> digestAlgorithmSet) {
+        this.digestAlgorithmSet = digestAlgorithmSet;
+    }
```

Comment:
Cosmetic: Code style.

```suggestion
    public Set<String> getDigestAlgorithmSet()
    {
        return digestAlgorithmSet;
    }

    public void setDigestAlgorithmSet(Set<String> digestAlgorithmSet)
    {
        this.digestAlgorithmSet = digestAlgorithmSet;
    }
```